src/react/Router.jsx

```diff
@@ -81,14 +81,17 @@ export default ({ reactHistory: history, astarteClient, sessionManager, config,
         <Route path="groups/new" element={<NewGroupPage {...pageProps} />} />
         <Route path="groups/:groupName/edit" element={<GroupDevicesSubPath {...pageProps} />} />
         <Route path="flows" element={<FlowInstancesPage {...pageProps} />} />
-        <Route path="flows/new/:pipelineId" element={<FlowConfiguration {...pageProps} />} />
-        <Route path="flows/:flowName" element={<FlowDetails {...pageProps} />} />
+        <Route
+          path="pipelines/:pipelineId/flows/new"
```

Comment:
this is a new flow, the path should start with flow (e.g. `/flows/new?pipelineId=<id>`)

Reply:
I agree. I checked documentation from Microsoft, Swagger and OpenAPI to look for best practices on REST routes; there are no strict guidelines here but the general approach is to keep routes as simple as possible, avoiding deep nesting.
I changed the route to `/flows/new?pipelineId=<id>` as you suggested.